xml/slemicro_supportconfig.xml

```diff
@@ -0,0 +1,956 @@
+    <para>
+      When performing the manual installation, <command>supportconfig</command>
+      is not available. However, you can collect log files from &yast; by using
+      <command>save_y2logs</command>. This command will create a
+      <filename>.tar.xz</filename> archive in the directory
+      <filename>/tmp</filename>.
+    </para>
+  </section>
+  <section xml:id="sec-admsupport-kernel">
+    <title>Support of kernel modules</title>
+    <para>
+      An important requirement for every enterprise operating system is the
+      level of support you receive for your environment. Kernel modules are the
+      most relevant connector between hardware (<quote>controllers</quote>) and
+      the operating system. Every kernel module in &productnameshort; has a
+      <literal>supported</literal> flag that can take three possible values:
+    </para>
+    <itemizedlist mark="bullet" spacing="normal">
+      <listitem>
+        <para>
+          <quote>yes</quote>, thus <literal>supported</literal>
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          <quote>external</quote>, thus <literal>supported</literal>
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          <quote/> (empty, not set), thus <literal>unsupported</literal>
+        </para>
+      </listitem>
+    </itemizedlist>
+    <para>
+      The following rules apply:
+    </para>
+    <itemizedlist mark="bullet" spacing="normal">
+      <listitem>
+        <para>
+          All modules of a self-recompiled kernel are by default marked as
+          unsupported.
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          Kernel modules supported by &suse; partners and delivered using
+          <literal>&suse; SolidDriver Program</literal> are marked
+          <quote>external</quote>.
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          If the <literal>supported</literal> flag is not set, loading this
+          module will taint the kernel. Tainted kernels are not supported.
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          Kernel modules not provided under a license compatible to the license
+          of the Linux kernel will also taint the kernel. For details, see the
+          state of <filename>/proc/sys/kernel/tainted</filename>.
+        </para>
+      </listitem>
+    </itemizedlist>
+    <section xml:id="sec-admsupport-kernel-background">
+      <title>Technical background</title>
+      <para/>
+      <itemizedlist mark="bullet" spacing="normal">
+        <listitem>
+          <para>
+            Linux kernel: The value of
+            <filename>/proc/sys/kernel/unsupported</filename> defaults to
+            <literal>2</literal> (<literal>do not warn in syslog when loading
+            unsupported modules</literal>). This default is used in the
+            installer and in the installed system.
```

Comment:
```suggestion
            <literal>2</literal>. This default is used in the installer
            and in the installed system (<literal>do not warn 
            in syslog when loading unsupported modules</literal>).
```

Reply:
Replacing parts of the sentence in an attempt to avoid an empty opening bracket in the line:
![image](https://user-images.githubusercontent.com/98878014/209122196-024efc67-93b2-43d3-9470-47699c19899b.png)